datascience/tables.py

```diff
@@ -1332,30 +1332,29 @@ def with_column(self, label, values):
 
         Args:
             ``label`` (str): The column label. If an existing label is used,
-                the existing column will be replaced in the new table.
+            the existing column will be replaced in the new table.
```

Comment:
No idea why it's complaining that the table isn't defined in the doc test... any ideas?